addons/stock_account/models/stock_valuation_layer.py

```diff
@@ -71,6 +72,15 @@ def action_open_layer(self):
             'views': [[False, "form"]],
             'res_id': self.id,
         }
+    # To get a journal of perticuler StockValuationLayer
+    def action_journal_entry(self):
+        self.ensure_one()
```

Comment:
When no account_move, it will open a from  to create. I don't think this is what we want. I guess return nothing if no account_move. Check with Thd about this.

Reply:
@yhu-odoo, my apologies for forgetting to hide this field when "account_move_id" is false. By hiding the field, we can prevent this problem from occurring.